docs/api.md

```diff
@@ -494,3 +494,34 @@ Output content of the table to a `pandas.DataFrame`.
 - `limit`: Optional[int]
     
     Maximum number of rows to return. Defaults to -1, i.e. all.
+
+
+## Custom metrics
+
+{% hint style="info" %}
+
+Custom metrics are an Enteprise feature. Get in touch at
+[support@sematic.dev](mailto:support@sematic.dev) to learn more.
+
+{% endhint %}
+
+### `log_metric`
```

Comment:
You should probably add the import path, as this doesn't come from `sematic.*` (unlike the other APIs here).